### PR TITLE
chore: Add verification that Hope approves the commit

### PR DESCRIPTION
**Pull request by DenisVerkhoturov**

### CODE REVIEW

File: hooks/pre-commit.d/check-hope-doe-approves.sh [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+cat << EOF
+--------------------------------------------------------------------------------
+-----██--██--██--██---████---█████---█████----████---██--██--██████--█████------
+-----██--██--██--██--██--██--██--██--██--██--██--██--██--██--██------██--██-----
+-----██--██--███-██--██--██--██--██--██--██--██--██--██--██--██████--██--██-----
+-----██--██--██-███--██████--█████---█████---██--██---████---██------██--██-----
+-----██████--██--██--██--██--██------██------██--██----██----██------██--██-----
+------████---██--██--██--██--██------██-------████-----██----██████--█████------
+--------------------------------------------------------------------------------
```
Comment (myazinn) [inline hooks/pre-commit.d/check-hope-doe-approves.sh]: I've noticed that there is a typo. "Unappoved" instead of "Unapproved". This is unacceptable!

Comment (DenisVerkhoturov) [reply hooks/pre-commit.d/check-hope-doe-approves.sh]: Sounds reasonable, I going to fix it in the next iteration! :thinking:

Comment (DenisVerkhoturov) [reply hooks/pre-commit.d/check-hope-doe-approves.sh]: @myazinn, Done, thank you for pointing that out.

**Comment by DenisVerkhoturov** [comment]
Just kidding! 🤣

**Pull request closed without merging**